### PR TITLE
Design feedback on images

### PR DESCRIPTION
**Pull request by oraNod**
Resolves issue #166 

- Increase the size and weight of text in the bullito welcome image.
- Redo the hero background image to better match the wireframe.
- Include all source and template files in the project.

### CODE REVIEW

**Comment by oraNod** [comment]
@tiyiprh Could you please take a look at these changes? We can increase the width and height of the bullito welcome image in the page template. We'll need to do that in a separate PR but I think the font weight looks nice at 500 for now.

File: images/welcome_to_ansible.svg
Comment (tiyiprh) [inline images/welcome_to_ansible.svg:1]: Is this using the RH Display font? It looks different to me.

File: themes/ansible-community/assets/images/unsplash.svg
Comment (tiyiprh) [inline themes/ansible-community/assets/images/unsplash.svg:1]: This looks good! I'm not sure if it's just my screen but it kind of looks like there are additional light lines on the rest of the background that make a kind of diamond pattern across the whole image...

File: images/welcome_to_ansible.svg
Comment (oraNod) [reply images/welcome_to_ansible.svg:1]: I couldn't get the RH display font in the tool I'm using, inkscape. I can try again but if you could provide an svg with the text in a speech bubble that'd be great.

File: themes/ansible-community/assets/images/unsplash.svg
Comment (oraNod) [reply themes/ansible-community/assets/images/unsplash.svg:1]: That sounds a bit off. I'm not sure.

https://ansible-community-website--167.org.readthedocs.build/en/167/assets/images/unsplash.jpg

Maybe it's resolution or a cache thing. From what I can tell the changes look pretty close to the wireframe. Could you send a screengrab?

Comment (tiyiprh) [reply themes/ansible-community/assets/images/unsplash.svg:1]: Strange, that link looks correct to me though!

File: images/welcome_to_ansible.svg
Comment (tiyiprh) [reply images/welcome_to_ansible.svg:1]: Ansibull

Here is an updated bullito image that includes the text bubble and text. Hopefully this helps with the issue! Sorry, for not sending the whole element as an svg earlier!

Comment (oraNod) [reply images/welcome_to_ansible.svg:1]: No worries! That looks so much better. I've added this to the PR and really love it. Thank you!

**Comment by oraNod** [comment]
Hmm. There seems to be an issue with the bullhorn subscribe image. Resizing the page it kind of gets constrained or just disappears. It's like I have to increase the size of the display by a lot to actually see that image.

@tiyiprh Do we also have an svg for that bullhorn subscribe mascot? Maybe the problem is the format (.png) but I'm only kind of guessing. Something we need to fix though.

**Comment by tiyiprh** [comment]
> Hmm. There seems to be an issue with the bullhorn subscribe image. Resizing the page it kind of gets constrained or just disappears. It's like I have to increase the size of the display by a lot to actually see that image.
> 
> @tiyiprh Do we also have an svg for that bullhorn subscribe mascot? Maybe the problem is the format (.png) but I'm only kind of guessing. Something we need to fix though.

Sorry, we didn't have that in SVG yet but we are working on creating an SVG for it. I will also send an updated SVG version of the bull in the contribute area but can probably create a new issue for that.

**Comment by oraNod** [comment]
@tiyiprh I've removed that commit to try and resize the bullhorn subscribe image and created a new issue so we can tackle that as a follow up: https://github.com/ansible-community/community-website/issues/176

**Comment by tiyiprh** [comment]
For the banner area, would it also be possible to make the bullito the same height as the content in the white box?

**Comment by oraNod** [comment]
Thanks for the bullhorn contribute svg @tiyiprh I've added that.

> For the banner area, would it also be possible to make the bullito the same height as the content in the white box?

I think we can but it's killing me right now. LOL. How about we add that to a separate issue and tackle it as part of the hackathon next week? Maybe we can solve it at the same time as https://github.com/ansible-community/community-website/issues/136

**Comment by GregSutcliffe** [comment]
I'm out of the loop on what needs to happen here vs other places, but the current render LGTM, ship it if it isn't waiting on anything.

**Review by GregSutcliffe — APPROVED** [review]

**Pull request merged**